pom.xml

```diff
@@ -187,4 +193,12 @@
 		</snapshotRepository>
 	</distributionManagement>
 
+	<dependencies>
+		<dependency>
+			<groupId>org.openmrs.module</groupId>
+			<artifactId>uiframework-api</artifactId>
+			<version>3.9</version>
+			<scope>provided</scope>
+		</dependency>
+	</dependencies>
```

Comment:
...also we could probably get rid of this too

Reply:
Fixed